README.md

```diff
@@ -38,7 +38,7 @@ packages, you will probably want the following:
 
 2. Activate the virtualenv.
 
-        source ./env/bin/activate
+        source ./venv/bin/activate
```

Comment:
`python -m venv env` creates a virtualenv called `env`, `venv` is the python module name.